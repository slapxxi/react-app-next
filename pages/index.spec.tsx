import { render } from 'react-testing-library';
import Index from './index';

it('works', () => {
  let { container } = render(<Index />);
  expect(container.firstChild).toMatchSnapshot();
});
