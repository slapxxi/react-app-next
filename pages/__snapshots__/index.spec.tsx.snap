// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
<div>
  <div>
    <div>
      <a>
        Home
      </a>
    </div>
    <div>
      <a>
        About
      </a>
    </div>
    <div>
      <a>
        Post
      </a>
    </div>
  </div>
  <div>
    <a>
      About
    </a>
  </div>
</div>
`;
